app/models/listing.rb

```diff
@@ -0,0 +1,10 @@
+class Listing < ActiveRecord::Base
+  include ConvertToFormattedTime
+  include WithUserAssociationExtension
+
+  attr_accessor :hour, :mins
+
+  belongs_to :user, dependent: :destroy
+
+  validates :start_time, numericality: { only_integer: true }, inclusion: (1..1440)
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)